btk/measure.py

```diff
@@ -143,33 +108,35 @@ def __init__(
     def __iter__(self):
         return self
 
-    def run_batch(self, blend_output, index):
-        deblend_results = self.measurement_params.get_deblended_images(
-            data=blend_output, index=index
-        )
-        measured_results = self.measurement_params.make_measurement(data=blend_output, index=index)
-        return [deblend_results, measured_results]
+    def run_batch(self, batch, index):
+        output = []
+        for f in self.measure_functions:
+            output.append(f(batch, index))
```

Comment:
I am unsure about which format to use exactly for the function. Shouldn't we only give it batch['blend_images'][index] ? Maybe adding the psf ? This does work so either way may be fine.

Reply:
I think as long as we are clear what the batch contains it should be ok for the user. I'm just giving the user all the info assuming that there might be different algorithms that require different info